src/components/com_tjcertificate/administrator/controllers/certificates.php

```diff
@@ -34,4 +38,100 @@ public function getModel($name = 'Certificate', $prefix = 'TjCertificateModel')
 	{
 		return parent::getModel($name, $prefix, array('ignore_request' => true));
 	}
+
+	/**
+	 * Method to delete the certificate.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function deleteCertificate()
+	{
+		$user = Factory::getUser();
+		$app  = Factory::getApplication();
+		$certificateId = $app->input->getInt('id');
+		$model = $this->getModel();
+
+		if (!$user->authorise('certificate.external.manage', 'com_tjcertificate'))
+		{
+			$checkCertificateExist = $model->checkCertificateExist($certificateId, $user->id);
+
+			if ($checkCertificateExist === false)
+			{
+				return false;
+			}
+		}
+
+		// Remove the items.
+		if ($model->delete($certificateId))
+		{
+			$this->setMessage(Text::_('COM_TJCERTIFICATE_CERTIFICATE_DELETED_SUCCESSFULLY'));
+		}
+
+		$this->setRedirect(Route::_('index.php?option=com_tjcertificate&view=certificates&layout=my&Itemid=' . $itemId, false));
```

Comment:
what will happen when you will delete from the admin side. There is no layout like `layout=my`

Reply:
this function is calling only from frontend